recipes/nvshmem/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "3.1.7" %}
```

Comment:
Why a separate recipe for this metapackage? Are there plans to add additional packages to the nvshmem umbrella?

Reply:
Technically, this package is redundant because you will get the same deps from installing libnvshmem-static.